packages/network/lib/cors.ts

```diff
@@ -193,9 +238,12 @@ export const urlMatchesPolicyBasedOnDomain = (frameUrl: string, topUrl: string):
  * @param topProps - The props of the url you are testing the policy in context of.
  * @returns boolean, true if matching, false if not.
  */
-export const urlMatchesPolicyBasedOnDomainProps = (frameUrl: string, topProps: ParsedHostWithProtocolAndHost): boolean => {
-  const obj = parseUrlIntoHostProtocolDomainTldPort(frameUrl)
-  const policy = strictSameOriginDomains.includes(`${obj.domain}.${obj.tld}`) ? 'same-origin' : 'same-super-domain-origin'
+export const urlMatchesPolicyBasedOnDomainProps = (frameUrl: string, topProps: ParsedHostWithProtocolAndHost, opts?: {
+  skipDomainInjectionForDomains: string[]
+}): boolean => {
+  const policy = policyForDomain(frameUrl, {
```

Comment:
Policy for domain check if `skipDomainInjectionForDomains` is an array. this should just pass through
```suggestion
  const policy = policyForDomain(frameUrl, opts)
```

Reply:
addressed in 8c6813dbf3